src/ClientBuilderInterface.php

```diff
@@ -108,4 +108,9 @@ public function setSerializer(SerializerInterface $serializer);
      * @return $this
      */
     public function setRepresentationSerializer(RepresentationSerializerInterface $representationSerializer);
+
+    /**
+     * @param string|null $sdkIdentifierSuffix The suffix for the SDK identifier, to be used by official SDK integrations
```

Comment:
Missing the description of what this method does (something can be taken from the description of the parameter that can then be reduced)